python/golem_task_api/server.py

```diff
@@ -91,13 +113,15 @@ def __init__(
         reply.discarded_subtask_ids.extend(discarded_subtask_ids)
         await stream.send_message(reply)
 
+    @forward_exceptions()
     async def RunBenchmark(self, stream):
-        request: RunBenchmarkRequest = await stream.recv_message()
+        _: RunBenchmarkRequest = await stream.recv_message()
```

Comment:
Do we need to assign the result at all if it's not used?
```suggestion
        await stream.recv_message()
```